src/Core/Translation/Util/Refresher.php

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
I'm sceptical about adding that new class because it does not seems to be used anywhere. Also, it uses some legacy and we try to avoid this in the `Core` namespace classes.

Reply:
> it does not seems to be used anywhere

I disagree with this argument. This should make module developer life easier, this is enough for this class to be acceptable.

Reply:
I understand it "might" make module devs life easier, but I agree with Fabien. This class is not used anywhere and also is not even tested. Doesn't sound good :zany_face: Maybe we could at least add some kind of tests?
I would like to get more opinions from other devs too.

Reply:
@zuk3975 PrestaShop is a tool for developers to create online stores, create modules, and themes. If the module developer says that this is something that is **required** to make translation work, then it's clear to me that we should either accept to solve the problem n the core translation mechanism to allow .xlf files in tabs, or we should accept a proposed solution to handle this scenario correctly.

Regarding the tests, the code is pretty straightforward but if you feel like it's safer to tests if it is really deleting the directories and returning translators, then ok :D